app/containers/TodoApp/components/TodoItem.js

```diff
@@ -0,0 +1,75 @@
+/* @flow */
+
+import React from 'react';
+import Relay from 'react-relay';
+import ListItem from '../../../components/ListItem';
+import RemoveTodoMutation from '../mutations/RemoveTodoMutation';
+import UpdateTodoMutation from '../mutations/UpdateTodoMutation';
+
+type Todo = {
+  id: string,
+  content: string,
+};
+
+type Props = {
+  todo: Todo,
+  user: {
+    id: string,
+  },
+  relay: {
+    commitUpdate: Function,
+  },
+};
+
+/**
+ * ListItem wrapper for TodoList implementation
+ * Parent component: TodoList
+ * Child component: ListItem
+ */
+class TodoItem extends React.Component {
+
+  props: Props;
+
+  // removeTodo function passed to the ListItem component
```

Comment:
these descriptions are fine, but i'd prefer if all the function descriptions have a standardized format